### PR TITLE
chore: Update ttl from long to duration

### PR DESCRIPTION
**Pull request by rishtigupta**
## PR description: 
- Update ttl from long to duration
- Refactor required integ tests

## Build:
- Ran `./gradlew :momento-sdk:spotlessApply && ./gradlew clean build` successfully

## Issue
#174

### CODE REVIEW

File: momento-sdk/src/main/java/momento/sdk/ScsDataClient.java [hunk 1]
```diff
@@ -486,19 +486,19 @@ private _DeleteRequest buildDeleteRequest(ByteString key) {
     return _DeleteRequest.newBuilder().setCacheKey(key).build();
   }
 
-  private _SetRequest buildSetRequest(ByteString key, ByteString value, long ttl) {
+  private _SetRequest buildSetRequest(ByteString key, ByteString value, Duration ttl) {
     return _SetRequest.newBuilder()
         .setCacheKey(key)
         .setCacheBody(value)
-        .setTtlMilliseconds(ttl)
+        .setTtlMilliseconds(ttl.getSeconds() * 1000)
```
Comment (cprice404) [inline momento-sdk/src/main/java/momento/sdk/ScsDataClient.java:493]: just curious - does the Duration object not have a `getMilliseconds()`?

**Review by cprice404 — COMMENTED** [review]
looks good other than a couple of minor nits!

File: momento-sdk/src/main/java/momento/sdk/ScsDataClient.java (hunk 1 above)
Comment (nand4011) [reply momento-sdk/src/main/java/momento/sdk/ScsDataClient.java:493]: Looks like it has a `toMillis()` method.

**Comment by nand4011** [comment]
I agree that this should go in before the SimpleCacheClient name change. Let the person with the wide reaching simple change do the rebase work.

**Review by nand4011 — APPROVED** [review]

**Pull request merged**